modules/installation-network-user-infra.adoc

```diff
@@ -48,6 +53,18 @@ the Cluster Version Operator on port `9099`.
 |`10256`
 |openshift-sdn
 
+
+.4+|UDP
+|`4789`
+|VXLAN and GENEVE
+
+|`6081`
+|VXLAN and GENEVE
+
+|`9000`-`9999`
+|Host level services, including the node exporter on ports `9100`-`9101` and
+the Cluster Version Operator on port `9099`.
```

Comment:
Let me check, but I think we should drop the CVO here.

Reply:
Confirmed 9099 since openshift/cluster-version-operator@0bb98c9e67b48b672fb947ea5728f66699deadfc , but it's just TCP for metrics, not this UTC entry.

Reply:
I'm dropping the CVO reference here. Thanks!